packages/lavapack/src/runtime-cjs-template.js

```diff
@@ -0,0 +1,85 @@
+;(function() {
+
+  // identify the globalRef
+  const globalRef = (typeof globalThis !== 'undefined') ? globalThis : (typeof self !== 'undefined') ? self : (typeof global !== 'undefined') ? global : undefined
+  if (!globalRef) {
+    throw new Error('Lavamoat - unable to identify globalRef')
+  }
+
+  // polyfill globalThis
+  if (globalRef && !globalRef.globalThis) {
+    globalRef.globalThis = globalRef
+  }
+
+  // polyfill node/browserify's globalRef
+  globalThis.global = globalThis
+
+  const {strictScopeTerminator} = templateRequire('strict-scope-terminator')
+
```

Comment:
everything below this line is the same as it was in the original `runtime-cjs.js` file

Reply:
While this applies equally to the existing `runtime.js`/`runtime-template.js` and due to consistency is probably out of scope for this PR, mixing source and built files in the `src` directory is a bit confusing. Maybe `runtime.js` and `runtime-cjs.js` could be in a separate directory?

In either case, what do you think of having `builder-runtime.js` prepend a comment with a message like below to the generated file?
```
// DO NOT EDIT! THIS FILE IS GENERATED FROM ./runtime-cjs-template.js BY ./builder-runtime.js
```